cmd/ebrelayer/txs/relay.go

```diff
@@ -0,0 +1,86 @@
+package txs
+
+// ------------------------------------------------------------
+//      Relay
+//
+//      Builds and encodes EthBridgeClaim Msgs with the
+//      specified variables, before presenting the unsigned
+//      transaction to validators for optional signing.
+//      Once signed, the data packets are sent as transactions
+//      on the Cosmos Bridge.
+// ------------------------------------------------------------
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtxb "github.com/cosmos/cosmos-sdk/x/auth/client/txbuilder"
+	amino "github.com/tendermint/go-amino"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/utils"
+
+	"github.com/swishlabsco/cosmos-ethereum-bridge/x/ethbridge"
+	"github.com/swishlabsco/cosmos-ethereum-bridge/x/ethbridge/types"
+)
+
+// RelayEvent
+//
+// Applies validator's signature to an EthBridgeClaim message containing information
+// 		about an event on the Ethereum blockchain before sending it to the Bridge
+//		blockchain. For this relay, the chain id (chainID) and codec (cdc) of the
+//		Bridge blockchain are required.
+//
+func RelayEvent(chainId string, cdc *amino.Codec, validatorAddress sdk.ValAddress, moniker string, passphrase string, claim *types.EthBridgeClaim) error {
+
+	cliCtx := context.NewCLIContext().
+		WithCodec(cdc).
+		WithAccountDecoder(cdc)
+
+	cliCtx = cliCtx.
+		WithFromAddress(sdk.AccAddress(validatorAddress)).
+		WithFromName(moniker)
+
+	cliCtx.SkipConfirm = true
+
+	txBldr := authtxb.NewTxBuilderFromCLI().
+		WithTxEncoder(utils.GetTxEncoder(cdc)).
+		WithChainID(chainId)
+
+	err := cliCtx.EnsureAccountExistsFromAddr(sdk.AccAddress(claim.ValidatorAddress))
+	if err != nil {
+		fmt.Printf("Validator account error: %s", err)
+	}
+
+	msg := ethbridge.NewMsgMakeEthBridgeClaim(*claim)
+
+	err1 := msg.ValidateBasic()
+	if err1 != nil {
+		fmt.Printf("Msg validation error: %s", err1)
+	}
+
+	cliCtx.PrintResponse = true
+
+	//prepare tx
+	txBldr, err = utils.PrepareTxBuilder(txBldr, cliCtx)
+	if err != nil {
+		fmt.Printf("Msg prepare error: %s", err)
```

Comment:
please append the string to the `err.Error()` instead (applies for this file)

Reply:
Fixed in next push https://github.com/cosmos/peggy/commit/28154a16ca1a6423a9726b6b72b4ab419bd01a9b